py_gtfs_rt_ingestion/tests/test_ingest.py

```diff
@@ -14,9 +21,64 @@ def test_argparse():
     assert args.input_file == dummy_file_path
     assert args.output_dir == dummy_output_dir
 
-def test_file_conversion():
+def test_file_conversion(tmpdir):
```

Comment:
using the `tmpdir` factory from pytest so that we can dump data and then have it cleaned up after running tests.